### PR TITLE
Feature/leaderboard filter

### PR DESCRIPTION
**Pull request by ue**

### CODE REVIEW

File: src/constants/options/leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export default ['day', 'week', 'month'];
+
+export const VALUE = ['Daily', 'Weekly', 'Montly'];
```
Comment (mistikk) [inline src/constants/options/leaderboard.js]: can you move to language file

File: src/providers/esteem/esteem.js [hunk 2]
```diff
@@ -131,7 +131,10 @@ export const addFavorite = (currentUsername, targetUsername) =>
 export const removeFavorite = (currentUsername, targetUsername) =>
   api.delete(`/favoriteUser/${currentUsername}/${targetUsername}`);
 
-export const getLeaderboard = () => api.get('/leaderboard').then(resp => resp.data);
+export const getLeaderboard = duration =>
+  api.get('/leaderboard', { params: { duration } }).then(resp => {
+    return resp.data;
+  });
```
Comment (mistikk) [inline src/providers/esteem/esteem.js]: can you add catch callback and you can send to bugsnag

File: src/constants/options/leaderboard.js (hunk 1 above)
Comment (feruzm) [reply src/constants/options/leaderboard.js]: Also make sure spelling is correct, “Monthly”, sublime has spell checker plugin if you use sublime 😉

**Comment by ue** [comment]
@mistikk updated

**Pull request merged**